chef/cookbooks/updater/recipes/default.rb

```diff
@@ -17,6 +17,9 @@
 # limitations under the License.
 #
 
+::Chef::Recipe.send(:include, CrowbarPacemaker::MaintenanceModeHelpers)
```

Comment:
Chef-ism. http://lists.opscode.com/sympa/arc/chef/2011-05/msg00286.html

Actually, I think that due the restructuring of the resources in my last update this is not required, I'll retry with the usual `include` to see